Basic Rules.txt

```diff
@@ -489,8 +489,8 @@ they cannot move, cannot stand under their own power, and fall prone if not
 supported by other means. They cannot speak, but can mumble incomprehensibly.
 Every turn that a character takes while they are downed, they must make a Death
 Saving Throw of DC 50. Success and failure of these throws is detailed above.
-When downed, any additional attack that hits regardless of damage is the same as
-having failed one Death Saving Throw.
+Even if you had 5 health and took 80 damage, when you go down you go to 0 health.
+Any additional attacks you sustain will push you into negative health. 

```

Comment:
Should you mention that there are no penalties for getting lots of negative health? up to you